src/systems/ec/specs/components/footer/docs/usage.md

```diff
@@ -1,3 +1,22 @@
 ## Usage
 
-Do's and don'ts
+The footer is located at the bootom of a page. It provides inforamtion from the European Commission. 
+
+### Main content area
+
+The main content area of the footer offers access to the political and the informational section of the European Commission. It also contain links to the European Union and the EU institutions.
+Users can also find information of copyright, the Commission's new web presence, resources for partners, cookies, legal notices, contact and social media.
```

Comment:
`information of` or `information about`?

Reply:
Hi Yannick, thanks for checking...grammarly doesn't work here. I will go through it again.